web-components/src/components/table/tokens/lm-table-tokens.js

```diff
@@ -7,23 +7,29 @@ const table = {
   component: "table",
   header: {
     "bg-color": {
-      light: colors.gray[10].name,
-      dark: colors.gray[80].name
+      light: colors.gray["05"].name,
+      dark: colors.gray["95"].name
```

Comment:
this swill work as `colors.gray[95].name`

Reply:
Set this variables according Table design: https://www.figma.com/file/wDbhcTS6fYP9wFyJ7VnE0r/Lumos-Components?node-id=11951%3A1